ts/features/lollipop/saga/index.ts

```diff
@@ -19,10 +24,20 @@ export function* generateLollipopKeySaga() {
     const newKeyTag = uuid();
     yield* put(lollipopKeyTagSave({ keyTag: newKeyTag }));
     yield* cryptoKeyGenerationSaga(newKeyTag, maybeOldKeyTag);
-    if (!lollipopLoginEnabled) {
-      // If the lollipop login is not enable we immediately delete
-      // the new generated key.
-      yield* deletePreviousCryptoKeyPair(O.some(newKeyTag));
+  } else {
+    try {
+      // If we already have a keyTag, we check if there is
+      // a public key tied with it.
+      yield* call(getPublicKey, maybeOldKeyTag.value);
+    } catch {
+      // If there is no key it could be for two reasons:
```

Comment:
Should we add an analytics event here, to track how many have a key tag but not a related key pair?